Source/drlg_l2.cpp

```diff
@@ -1,5 +1,6 @@
 #ifndef SPAWN
 
+#include <cwchar>

```

Comment:
This is a reminder to myself to check if this is still broken in the latest version of the crosstools

Reply:
Looks like this is fixed now

Reply:
@glebm You sure? :) I am building a new docker image with the cross tools so I can check

Reply:
@glebm Correct, it is fixed! =)